setup.py

```diff
@@ -19,7 +19,7 @@
     install_requires.append('greenlet')
 
 setup(name='neovim',
-      version='0.1.10',
+      version='0.1.11dev',
```

Comment:
No, `setup.py` can't import the package currently being installed. If anything, information should flow in the other direction, but I don't know how...